### ISSUE TITLE
Investigate `Object.hasOwn` not working

### ISSUE THREAD

**Issue by dsherret** [denoland/dnt#34]
It has been in Node.js since 16.9:

```
npm/src/deps/deno_land/std_0.114.0/ws/mod.ts:14:9 - error TS2339: Property 'hasOwn' does not exist on type 'ObjectConstructor'.

14 const { hasOwn } = Object;
           ~~~~~~
```

**Comment by dsherret** [comment]
This is because `Object.hasOwn` is in ES2022 https://github.com/microsoft/TypeScript/blob/ae582a22ee1bb052e19b7c1bc4cac60509b574e0/src/lib/es2022.object.d.ts

But right now be default we are using a target of ES2021 for Node 16 as pointed out here: https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping

**Comment by dsherret** [comment]
Fixed in 69b8a163951074cf9f4b7b8beff173d7319ca695 (I accidentally pushed it to main, which I've now disabled for admins)
